st2common/tests/unit/test_param_utils.py

```diff
@@ -362,15 +362,17 @@ def test_get_finalized_params_with_list(self):
             'a2': 'Test',
             'a3': 'Test2',
             'a4': '{{a1}}',
-            'a5': ['{{a2}}', '{{a3}}']
+            'a5': ['{{a2}}', '{{a3}}'],
+            'a6': [['{{r1}}', '{{a2}}'], ['{{a3}}', '{{a1}}']]
```

Comment:
I would also note here that my reason for fixing this (expect runner) just needs this to work for strings. I'd just like this to be as clean as possible for future uses.